test/logr.test.js

```diff
@@ -37,6 +37,18 @@ describe('logr', () => {
       log(['tag1', 'tag2'], 'message');
       expect(lastMessage).to.contain('[tag1,tag2] message');
     });
+    it('allows you to specify a tag to trigger a ding', () => {
```

Comment:
also a test case to make sure it isn't added if it doesn't match.  there is already an indirect test, but would be nice to have an explicit one